src/test/java/uk/gov/hmcts/reform/sscs/ccd/domain/HearingDurationTest.java

```diff
@@ -0,0 +1,72 @@
+package uk.gov.hmcts.reform.sscs.ccd.domain;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.List;
+import org.junit.Test;
+
+public class HearingDurationTest {
```

Comment:
These test names and test data don't really explain anything that's going on in these tests - would be hard for something to understand the logic based on these tests i.e me lol.

Up to you if you want to change it or leave as for now.